src/components/CommentsList.jsx

```diff
@@ -29,25 +29,23 @@ function CommentsList (){
             )
     } else {
         return(
-            <div>
                 <div>
                     <p className="big-comments">Comments:
                     </p> 
                     {comments.map((comment) => {
                         return (
                         <>
-                        <div className="comment-div">
-                            <p className="p-author">{comment.author}</p>
-                            <p className="p-body">{comment.body}</p>
+                        <div key="comment-key" className="comment-div">
+                            <p key="author-key" className="p-author">{comment.author}</p>
```

Comment:
Only need the key on the div. everything else is wrapped in the div tag, so don't need the key